### ISSUE TITLE
no ammo: we can hear “click” sound  although the human model does not pull the trigger

### ISSUE THREAD

**Issue by illwieckz** [Unvanquished/Unvanquished#565]
Hi, if you have sufficient ammo when you fire,

- the finger pull the trigger
- the weapon firing sound is heard
- the projectile is fired

If you have not sufficient ammo when you attempt to fire,

- the finger does not pull the trigger
- the “missing ammo” click sound is heard

So it's not realistic at all, since the “missing ammo” click sound can't be heard if the trigger is not pulled.

This bug is really not critical since very little visible. :blush:

**Comment by Viech** [comment]
This is a mocky response to [this](https://forums.unvanquished.net/viewtopic.php?f=44&p=15382#p15382), isn't it? :stuck_out_tongue_closed_eyes:

**Comment by illwieckz** [comment]
Yes but it is a real issue, I have some more if you want. :stuck_out_tongue:

**Comment by Viech** [comment]
I don't think we'll want to spend any resources on splitting weapon fire animations before beta.
